gidgethub/httpx.py

```diff
@@ -1,13 +1,16 @@
 import asyncio
 from typing import Mapping, Tuple, Any
 
-import httpx
+try:
+    from httpx import AsyncClient as Client
+except ImportError:
```

Comment:
We only support starting at Python 3.6, so...

```suggestion
except ModuleNotFoundError:
```